templates/zerver/help/logging-in.md

```diff
@@ -13,27 +13,34 @@ how you signed up. E.g. if you originally signed up using your Google
 account, you can later log in using GitHub, as long as your Google account
 and GitHub account use the same email address.
 
-
 ### Log in to a Zulip organization for the first time
 
 {start_tabs}
 
-{tab|desktop}
+{tab|web}
+
+1. Go to your organization's **Zulip URL**.
+
+1. Follow the on-screen instructions.
 
-1. Open the **left sidebar** (`Ctrl` + `Shift` + `s`).
+{tab|desktop}
 
-1. Set your [proxy settings](/help/connect-through-a-proxy) or add a
-   [custom certificate](/help/custom-certificates) if needed (rare).
+!!! warn ""
+    If you are having trouble connecting, you may need to set your
+    [proxy settings](/help/connect-through-a-proxy) or add a
+    [custom certificate](/help/custom-certificates).
 
-1. Click the **plus** (+) icon.
+1. Click the **plus** (<i class="fa fa-plus"></i>) icon on the far left side of the app. You can also click on **Zulip** > **Add organization** in the main menu bar.
 
-1. Enter your Zulip URL, and click **Connect**.
+1. Enter your Zulip URL and click **Connect**.
```

Comment:
Hm, what's the general pattern regarding  Oxford commas in the help center? I'm not sure we want to drop it.